app/actions/GroupActions.ts

```diff
@@ -32,6 +32,27 @@ export function addMember({ groupId, userId, role }: AddMemberArgs) {
   });
 }
 
+export const deleteMembershipHistory = ({
+  groupId,
+  userId,
+}: {
+  groupId: EntityId;
+  userId?: EntityId;
+}) => {
+  return callAPI({
+    types: MembershipHistory.DELETE,
+    endpoint: `/membership-history`,
```

Comment:
```suggestion
    endpoint: `/membership-history/${groupId}`,
```
It's more common to follow this schema when doing delete methods, instead of including the id in the body